tests/importer/model_flattening.cpp

```diff
@@ -832,3 +830,132 @@ TEST(ModelFlattening, importedComponentsWithConnectionsToChildren)
     auto a = printer->printModel(modelUsingImports);
     EXPECT_EQ(e, a);
 }
+
+TEST(ModelFlattening, resolveFlattenCircularImportsComponents)
+{
+    std::string resolveError = "Cyclic dependencies were found when attempting to resolve components in model 'importExample2b'. The dependency loop is:\n"
```

Comment:
Should be made `const` (same elsewhere in this PR).

Reply:
a107cfd4

Reply:
1960e45a